defaults/main.yml

```diff
@@ -16,6 +16,11 @@ mysql_config_file: /etc/mysql/my.cnf
 mysql_pid_file: /var/run/mysqld/mysqld.pid
 mysql_datadir: /var/lib/mysql
 
+# Allowing playbooks to provide external option files
+mysql_extra_conf_include_dir: /etc/mysql/conf.d/
```

Comment:
i'wd remove the trailing slash, to avoid ugly paths like: "{{ mysql_extra_conf_include_dir }}/test_template.cnf":

Reply:
done 😄